lib/registry.js

```diff
@@ -27,10 +28,12 @@ class Registry {
 		const type = ['#', 'TYPE', name, item.type].join(' ');
 
 		const values = (item.values || []).reduce((valAcc, val) => {
-			const labels = Object.keys(val.labels || {}).map(
+			let labels = Object.keys(val.labels || {}).map(
 				key => `${key}="${escapeLabelValue(val.labels[key])}"`
 			);
 
+			labels = Object.assign(this._defaultLabels, labels);
```

Comment:
if you keep `_defaultLabels` as an object internally, then do `Object.assign` before the `map`?